src/columns.rs

```diff
@@ -190,4 +192,13 @@ mod tests {
     fn wrap_columns_panic_with_zero_columns() {
         wrap_columns("", 0, 10, "", "", "");
     }
+
+    #[test]
+    fn wrap_columns_with_tabs() {
+        let options = Options::new(23).tab_width(4);
+        assert_eq!(
+            wrap_columns("hello\tthis\tis\tlong\tyeah", 2, options, ".", ".", "."),
+            vec![".hello     .is\tlong.", ".this      .yeah      ."]
```

Comment:
The `.`s here confused me :smile: Could you use `|` or something similar instead so that it doesn't look like it's part of the text?

Reply:
That's fair, `'.'` is a bit unexpected for a column divider. I've changed that in 324ba420b4aad16860ef74fbff6fd4265b82b91c.